### PR TITLE
[ADD] calendar employee double booking

### PR DESCRIPTION
**Pull request by rimbendhaou**

### CODE REVIEW

File: project_resource_calendar/views/res_partner_view.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_partner_inherit_form_view_2" model="ir.ui.view">
```
Comment (jananjoy) [inline project_resource_calendar/views/res_partner_view.xml:3]: not a good record Id  name 'view_partner_inherit_form_view_2' . Besides in the previous record id = view_employee_form_inherited and here different naming convention

File: project_resource_calendar/__manifest__.py [hunk 2]
```diff
@@ -34,6 +34,8 @@
         'views/resource_view.xml',
         'views/room_view.xml',
         'views/instrument_view.xml',
+        'views/res_partner_view.xml',
```
Comment (jananjoy) [inline project_resource_calendar/__manifest__.py:37]: @rimbendhaou :Is dependency required or not for  res.partner  in the manifest file (reference in project event )

File: project_resource_calendar/models/hr_employee.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+    allow_double_book = fields.Boolean(
+        string='Allow Double Booking',
+        help='Check if this employee '
+             'can be booked as a participant in more than '
+             'one meeting or event at the same '
+             'time.',
+        readonly=True,
+        related='user_id.partner_id.allow_double_book',
+    )
```
Comment (jananjoy) [inline project_resource_calendar/models/hr_employee.py:18]: @rimbendhaou  Add an extra line at the end as standard format

File: project_resource_calendar/models/res.partner.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    _inherit = "res.partner"
+
+    allow_double_book = fields.Boolean(
+        string='Allow Double Booking',
+        help='Check if this employee '
+             'can be booked as a participant in more than '
+             'one meeting or event at the same '
+             'time.',
+        default=True,
+    )
```
Comment (jananjoy) [inline project_resource_calendar/models/res.partner.py:17]: @rimbendhaou : Add extra line at the end of the file

File: project_resource_calendar/views/hr_employee_view.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Employee -->
+        <record id="view_employee_form_inherited" model="ir.ui.view">
```
Comment (jananjoy) [inline project_resource_calendar/views/hr_employee_view.xml:4]: @rimbendhaou No uniform name convention with other  files in the project . Hope need to define a standard naming conventions for the whole project

**Review by jananjoy — CHANGES_REQUESTED** [review]

File: project_resource_calendar/models/hr_employee.py (hunk 3 above)
Comment (jananjoy) [reply project_resource_calendar/models/hr_employee.py:18]: @rimbendhaou Have you add the corresponding test cases to validate this variable.

Comment (rimbendhaou) [reply project_resource_calendar/models/hr_employee.py:18]: @jananjoy It is a work in progress PR (not finished yet). I still work on it.

**Pull request closed without merging**